src/form-view.js

```diff
@@ -109,7 +109,12 @@ const viewMethods = {
     let iconsInput = document.createElement('atom-text-editor');
     iconsInput.setAttribute('mini', true);
     let iconsList = document.createElement('div');
-    iconsList.classList.add('block', 'icons-list');
+    let iconsListStyle = atom.config.get('project-viewer.iconListStyle');
```

Comment:
All rewrite this part then, even though after i add the `onDidChange` it is not constant and therefore a `let` ;)